x-pack/legacy/plugins/siem/server/lib/detection_engine/routes/tags/read_tags_route.ts

```diff
@@ -5,13 +5,14 @@
  */
 
 import Hapi from 'hapi';
-import { isFunction } from 'lodash/fp';
+
 import { DETECTION_ENGINE_TAGS_URL } from '../../../../../common/constants';
-import { ServerFacade, RequestFacade } from '../../../../types';
+import { LegacyServices, LegacyRequest } from '../../../../types';
 import { transformError } from '../utils';
 import { readTags } from '../../tags/read_tags';
+import { GetScopedClients } from '../../../../services';
 
-export const createReadTagsRoute: Hapi.ServerRoute = {
+export const createReadTagsRoute = (getClients: GetScopedClients): Hapi.ServerRoute => ({
```

Comment:
Just throwing this out there -> we are mixing `({})` and `{ return {} }` in between the routes. Not sure if we want to take this opportunity to create some consistency?

Reply:
Yeah, I prefer the former unless there's a good reason. In the interest of this PR, the reason was not introducing a lot of whitespace changes: if converting caused everything to be re-indented, I didn't do it.